azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
```

Comment:
I was thinking "additional" DDL only. So the column would be altered for h2 when it's initialized for the first time. I would avoid any migrations required on existing installations on MySQL databases as long as possible.